seqr/views/utils/project_context_utils.py

```diff
@@ -97,7 +97,8 @@ def _add_parent_ids(response, projects, family_models, individual_models, locus_
     prefetch_related_objects(locus_lists_models, 'projects')
     for locus_list in locus_lists_models:
         for project in locus_list.projects.all():
-            response['projectsByGuid'][project.guid]['locusListGuids'].append(locus_list.guid)
+            if project.guid in response['projectsByGuid']:
+                response['projectsByGuid'][project.guid]['locusListGuids'].append(locus_list.guid)
```

Comment:
Will the 'locusListGuids' always in `response['projectsByGuid'][project.guid]`?

Reply:
yes, immediately above this are the line
```
    for project in response['projectsByGuid'].values():
        project['locusListGuids'] = []
```